### PR TITLE
Allow invoices to be sent/resent from the invoice view page

### PR DESCRIPTION
**Pull request by mwikala**
This fixes issue #404 by allowing a user to resend an invoice on the invoice view page.

### CODE REVIEW

**Comment by mohitpanjwani** [comment]
We are not showing this option on view page because there's already a button above to send the invoice. 

I think we should only show the resend option (not send).

**Comment by mwikala** [comment]
> We are not showing this option on view page because there's already a button above to send the invoice. 

In the new design there is no button to send or resend the invoice on the pdf preview page

**Comment by mohitpanjwani** [comment]
CleanShot 2021-12-15 at 13 28 29@2x

You don't see this button?

File: resources/scripts/components/dropdowns/InvoiceIndexDropdown.vue [hunk 1]
```diff
@@ -153,15 +153,13 @@ const utils = inject('utils')
...
     (row.status == 'SENT' || row.status == 'VIEWED') &&
-    route.name !== 'invoices.view' &&
     userStore.hasAbilities(abilities.SEND_INVOICE)
   )
 }
 
 function canSendInvoice(row) {
   return (
     row.status == 'DRAFT' &&
-    route.name !== 'invoices.view' &&
```
Comment (mohitpanjwani) [inline resources/scripts/components/dropdowns/InvoiceIndexDropdown.vue:164]: @mwikala If you revert this change, I can merge it.

**Comment by mwikala** [comment]
@mohitpanjwani That should be ready now. Thanks

**Comment by mohitpanjwani** [comment]
Thanks 👌

**Pull request merged**